### ISSUE TITLE
Resolve revisit profile URL

### ISSUE THREAD

**Issue by MohammedElsayyed** [iipc/warc-specifications#18]
The following refers to a page not found:
http://netpreserve.org/warc/1.0/revisit/identical-payload-digest
http://netpreserve.org/warc/1.0/revisit/server-not-modified

I am not pretty sure whether they should be resolved or not.

**Comment by kris-sigur** [comment]
Those URIs serve as identifiers and the fact that they happen to the valid URLs is immaterial to their use in the WARC spec.

That said, we could have them resolve to something if we care to. Perhaps set up a page here and have netpreserve.org redirect to it.

**Comment by kris-sigur** [comment]
Actually, I think we already have the right things to redirect to;

https://iipc.github.io/warc-specifications/specifications/warc-format/warc-1.0/#profile-identical-payload-digest
https://iipc.github.io/warc-specifications/specifications/warc-format/warc-1.0/#profile-server-not-modified

**Comment by MohammedElsayyed** [comment]
Yeah, as @anjackson has said before, they are used as namespaces, so they don't necessarily have to resolve. 

I see it would be good to redirect them to the links above.
